samples/snippets/src/main/java/com/example/speech/SpeechProfanityFilter.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.speech;
+
+// [START speech_transcribe_with_profanity_filter_gcs]
+import com.google.cloud.speech.v1.RecognitionAudio;
+import com.google.cloud.speech.v1.RecognitionConfig;
+import com.google.cloud.speech.v1.RecognitionConfig.AudioEncoding;
+import com.google.cloud.speech.v1.RecognizeResponse;
+import com.google.cloud.speech.v1.SpeechClient;
+import com.google.cloud.speech.v1.SpeechRecognitionAlternative;
+import com.google.cloud.speech.v1.SpeechRecognitionResult;
+import java.io.IOException;
+import java.util.List;
+
+public class SpeechProfanityFilter {
+
+  public void speechProfanityFilter() throws Exception {
+    String uriPath = "gs://cloud-samples-tests/speech/brooklyn.flac";
```

Comment:
I think other samples use the style of adding a TODO for the developer.
```suggestion
    // TODO(developer): provide a gs:// path to your test input
    String uriPath = "gs://cloud-samples-tests/speech/brooklyn.flac";
```